src/sagas/messages/sendMessage/sendMessage.saga.ts

```diff
@@ -89,14 +66,13 @@ export function* sendMessageSaga(
     createdAt: currentTime,
     signature,
     pubKey,
-    channelId: channel,
+    channelId: channelAddress,
```

Comment:
@vinkabuki I guess it can be removed from iMessage interface as channelAddress is already included in this method payload (shouldn't we add ticket in waggle?)